src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/DefaultGetCaseViewOperation.java

```diff
@@ -48,12 +49,14 @@ public DefaultGetCaseViewOperation(@Qualifier(CreatorGetCaseOperation.QUALIFIER)
                                        UIDService uidService,
                                        ObjectMapperService objectMapperService,
                                        CompoundFieldOrderService compoundFieldOrderService,
-                                       FieldProcessorService fieldProcessorService) {
+                                       FieldProcessorService fieldProcessorService,
+                                       EnablingConditionParser enablingConditionParser) {
```

Comment:
should we be adding these java changes, I also see sql changes related to event enabling too?

Reply:
no you are right we shouldn't be adding java code changes, this must have slipped in when I was cherrypicking some changes across, will remove them from the pr